### ISSUE TITLE
String length not initialized

### ISSUE THREAD

**Issue by gregerts** [ttsiodras/asn1scc#186]
I am testing the master branch after having used an older release, and notice that the nCount of string does not seem to be initialized any more.

The type:

```
maxSize-T-String T-UInt32 ::= 256

T-String ::= OCTET STRING (SIZE(0..maxSize-T-String))
```
Produces the following initialization code:
```
void T_String_Initialize(T_String* pVal)
{
	memset(pVal->arr, 0x0, 256);
}
```
This fails to encode if the nCount is not explicitly set to 0. Is this the intended behaviour?

**Comment by maxime-esa** [comment]
Confirmed, I also checked in Ada, it does set the value of Length but it sets it to 256 instead of 0

![image](https://user-images.githubusercontent.com/1416005/57072011-8024a180-6cdc-11e9-9bbd-a68b82d2ef2e.png)

**Comment by usr3-1415** [comment]
Issue fixed in the latest commit (version 4.1.0.11f)
